includes/html/pages/oxidized.inc.php

```diff
@@ -82,7 +82,7 @@
         formatters: {
             "hostname": function(column, row) {
                 if (row.id) {
-                    return '<a href="/device/device=' + row.id + '">' + row.hostname + '</a>';
+                    return '<a href="device/device=' + row.id + '">' + row.hostname + '</a>';
```

Comment:
Shouldn't the `url()` function be used here ?

Reply:
The new URL format is `device/<device_id>`  such as `https://librenms.url/device/1`